pkg/kindsys/kindcats.cue

```diff
@@ -100,6 +100,28 @@ Core: S=close({
 
 	lineage: { name: S.machineName }
 	lineageIsGroup: false
+
+	// crd contains properties specific to converting this kind to a Kubernetes CRD.
+	crd: {
```

Comment:
this seems to be missing the `description` added in https://github.com/grafana/grafana/pull/60207: https://github.com/grafana/grafana/blob/4567b502f23a03ef5ba69f642e4cf277b1255ca8/pkg/kindsys/kindcats.cue#L83-L88